File: iosApp/iosApp/feature/cafe/cafe_list/CafeItem.swift
//
//  CafeItem.swift
//  PapaKarloSwift
//
//  Created by Марк Шавловский on 09.03.2022.
//
import Foundation
import shared
import SwiftUI

struct CafeItem: Identifiable {
    let id: String
    let address: String
    let phone: String
    let workingHours: String
    let cafeOpenState: CafeOpenState
    let latitude: Float
    let longitude: Float
}

File: shared/src/commonMain/kotlin/com/bunbeauty/shared/presentation/cafe_list/CafeItem.kt
package com.bunbeauty.shared.presentation.cafe_list

import com.bunbeauty.shared.domain.model.cafe.CafeOpenState

data class CafeItem(
    val uuid: String,
    val address: String,
    val phone: String,
    val workingHours: String,
    val cafeOpenState: CafeOpenState,
    val isLast: Boolean,
)
